src/kepler_model/train/profiler/node_type_index.py

```diff
@@ -34,6 +39,8 @@ def rename(name: str) -> str:
 
 
 def format_processor(processor):
+    if len(processor) < 2: # brand_raw is set to "-" on some machine
```

Comment:
minor: do you want to call it `processors` to indicate plural and also `type hint` as `list[str]` ?

Reply:
processor is string. This line to avoid the case that I found the processor cannot be retrieved since its value is "-".

Reply:
how about we type hint :str. ?